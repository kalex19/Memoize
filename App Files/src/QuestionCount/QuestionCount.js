import React, { Component } from 'react';
import Style from './QuestionCount.scss';

export default class QuestionCount extends Component {
  constructor(props){
  super(props);
  this.state = {

  }
}

render () {
    return(
            <div className={this.props.start ? "App-hidden" : "Question-count"}>
                <h1 className="Question-count-text">QUESTION #</h1>
                <p className="Question-count-text">{this.props.questionCount} / 30</p>
            </div>
        )
    }
}