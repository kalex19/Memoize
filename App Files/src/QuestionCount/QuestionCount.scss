@import '../variables.scss';

.Question-count {
    display: block; 
    height: 15vh;
    width: 25%;
    margin: 0 2%;
    border: dashed 1px $main-color;
}

.Question-count-text {
  font-size: 3vh;
  color: $main-color;
}

.App-hidden {
  visibility: hidden;
  width: 40%;
  margin: 0;
}

@media screen and (max-width: 414px) {
  .Question-count {
    width: 60%;
  }
}