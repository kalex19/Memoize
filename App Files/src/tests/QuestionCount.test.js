import React from 'react';
import QuestionCount from '../QuestionCount/QuestionCount.js';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('QuestionCount', () => {
    let wrapper;
  
    beforeEach(() => {
      wrapper = shallow(
        <QuestionCount
        />
      );
    });

    it('should match the snapshot with all data passed in', () => {
        expect(wrapper).toMatchSnapshot();
      });

  });