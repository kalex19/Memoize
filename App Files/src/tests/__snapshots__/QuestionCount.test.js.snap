// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionCount should match the snapshot with all data passed in 1`] = `
<div
  className="Question-count"
>
  <h1
    className="Question-count-text"
  >
    QUESTION #
  </h1>
  <p
    className="Question-count-text"
  >
     / 30
  </p>
</div>
`;
